src/js/content.js

```diff
@@ -64,6 +62,14 @@ var _jQuery = $.noConflict(true);
       dialogPosition: 'bottom'
     });
 
+    var innerHeight = window.innerHeight;
```

Comment:
This is needed so that `jq-scalebreaker` will move its popup when the native popover is opened for choosing the language. There is apparently no way to detect that the popover has been opened or (especially) closed without polling (see http://stackoverflow.com/questions/30671194/detecting-when-native-select-widget-is-opened-closed-in-uiwebview).